src/api/client/controllers.js

```diff
@@ -0,0 +1,122 @@
+const clientService = require('../../services/client');
+
+const getAllClients = async (req, res) => {
+  try {
+    const clients = await clientService.findAll();
+    res.status(200).send(clients);
+  } catch (e) {
+    console.log(e);
+    res.sendStatus(500);
+  }
+};
+
+const addClient = async (req, res) => {
+  if (!req.body || !req.body.companyName) {
+    return res.status(400).send('Project name cannot be empty');
+  }
+
+  const { isAdmin } = req;
+  if (!isAdmin) {
+    return res.status(403).send('Forbidden');
+  }
+  const { companyName, ...optionalFields } = req.body;
+  const client = await clientService.findOne({ companyName });
+
+  if (client) {
+    return res.status(409).send('Client with this name already exsists');
+  }
+
+  const payload = {
+    companyName,
+    ...optionalFields,
+  };
+
+  try {
+    const createdClient = await clientService.addNewClient(payload);
+    const { ...retVal } = createdClient.dataValues;
```

Comment:
Is this the same as this :)

```suggestion
    const retVal = createdClient.dataValues;
```

Reply:
I was thinking in way if i should hide something in response (like password in user), but after that i removed that part and forgot to change. I will fix it. Thanks.